_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,190 @@
+rule_type: atomic
+description: |
+  This rule checks that there are mechanisms to pause, stop or hide the auto-updating of text content.
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.2.2: # Pause, Stop, Hide (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgements:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [visible][] [HTML element][] in an [HTML document][] if:
+
+- the `innerText` property of the [element][html element] changes without the user [activating][activate] or [focusing][focus] any [element][html element] in the same [HTML document][];
+- the [element][html element] does not have [children][child] whose `innerText` property is also changed;
+- the change happens after the [readiness][document readiness] of the [HTML document][] the [element][html element] belongs to is equal to "complete";
+- it is not the only [content][] in the [HTML document][].
+
+## Expectation
+
+For the test target a [user interface component][] is provided to pause, stop or hide the change of the [text content][].
+
+**Note**: If there is more than one test target, a single [user interface component][] may be used to pause, stop or hide changing all test targets.
+
+## Assumptions
+
+- This rule assumes that the auto-updating of the content is not [essential][], which is listed as valid exception to [Success Criterion 2.2.2: Pause, Stop, Hide][sc 2.2.2]. When the auto-updating of content is [essential][] this rule may produce incorrect results.
+- This rule assumes that any [content][] changes are enabled by the content of [HTML document][] the test target belongs to. Changes originated by any other sources (e.g. browser shortcuts, browser extensions, browser settings, user agents, external browser applications) are not considered.
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+- [Understanding Success Criterion 2.2.2: Pause, Stop, Hide][sc 2.2.2]
+- [G186: Using a control in the Web page that stops moving, blinking, or auto-updating content][g186]
+- [F16: Failure of Success Criterion 2.2.2 due to including scrolling content where movement is not essential to the activity without also including a mechanism to pause and restart the content][f16]
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The text content automatically updates after the page completes loading. A button is available to stop the automatic updates.
+
+```html
+<body onload="startUpdates()">
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" onclick="stopUpdates()" value="Stop updates" />
+
+	<p>
+		The W3C Web Accessibility Initiative (WAI) develops standards and support materials to help you understand and
+		implement accessibility.
+	</p>
```

Comment:
I am not sure this is actually necessarily (in any of the examples).
There is already the "Random number:" part which makes the auto-updating `target` **not** the only content in the page. (and Inapplicable example 3 does not have it…)

Reply:
This still stands. I think all examples can be trimmed of this `p` and made shorter and simpler.